tests/transaction.rs

```diff
@@ -81,11 +75,8 @@ async fn qldb_transaction_rollback() -> Result<()> {
 
                 let count = result?;
 
-                let count = match &count[0] {
-                    IonValue::Struct(data) => match data.get("_1") {
-                        Some(IonValue::Integer(count)) => count,
-                        _ => panic!("First count returned a non integer"),
-                    },
+                let count = match &count[0].get("_1") {
+                    Some(IonValue::Integer(count)) => count,
```

Comment:
This code is the same as in lines 112, 113, maybe it could be a function to get a count.

Reply:
Yep, and it isn't needed because now we have the method `.count()`. I will replace by it.